internal/plugin/validators/lua_validator.go

```diff
@@ -354,7 +354,7 @@ func (v *LuaValidator) getDB(ctx context.Context) (store.Store, error) {
 	}
 	reqCluster, ok := ctx.Value(util.ContextKeyCluster).(*grpcModel.RequestCluster)
 	if !ok {
-		return nil, errors.New("invalid context: failed to retrieve RequestCluster from context")
+		reqCluster = &grpcModel.RequestCluster{Id: "default"}
```

Comment:
We really should store `default` as a constant in the codebase & use that across everything. Don't want to make that change in this PR, though perhaps we should track some work for the change.

Reply:
Implemented in https://github.com/Kong/koko/pull/482.